src/Voxel.MiddyNet/MiddyNetContext.cs

```diff
@@ -35,5 +37,11 @@ public void AttachToLambdaContext(ILambdaContext context)
         }
 
         public Func<ILambdaLogger, IMiddyLogger> LoggerFactory { get; }
+
+        internal void FinishedBeforeMiddlewares()
+        {
+            MiddlewareAfterExceptions = new List<Exception>();
```

Comment:
Why we do the initialization here and not in the constructor?

Reply:
Because I don't want users to be able to modify the collection, I initialize it as null.